Little Changes

xElkomy opened the pull request:
```
Change default.yaml and replace -impact with -severity
```

2 -
```
Add to the requirements.txt werkzeug==2.0.0 because the new versions make an errors when try to run the server.
```

And You must download the new version of nuclei to change the -impact to -severity but in the old version still use -impact.

xElkomy committed:
Update default.yaml

xElkomy committed:
Update requirements.txt

BitTheByte commented:
Hi @xElkomy 

Your work is really appreciated. I'm working on more clean and stable version of the tool which should resolve this issue. Should be ready to release in a week or two. Meanwhile I'll keep this PR open

Again, Thanks for your contribution and taking time to improve this codebase

xElkomy committed:
Some Changes

Update the directories and make install file to install the thirdparties tools instead of leave it on the directory always without update it

xElkomy committed:
Merge branch 'master' of https://github.com/xElkomy/Monitorizer
